webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-summary.js

```diff
@@ -302,14 +322,20 @@ export class SakaiRubricSummary extends rubricsApiMixin(RubricsElement) {
     return Math.sqrt(total).toFixed(2);
   }
 
-  _getCustomCount(criterionId, floorPoints) {
+  _getCustomCount(criterionId, floorPointsParam) {
 
     let ceilingPoints = 5000;
+    let floorPoints = parseInt(floorPointsParam);
     const criterion = this.criteria.find(c => c.id === criterionId);
+    let weightNow = 1;
+    if (this.rubric.weighted) {
+      weightNow = criterion.weight / 100;
+      floorPoints = floorPoints * weightNow;
+    }
```

Comment:
```suggestion
    const criterion = this.criteria.find(c => c.id === criterionId);
    const weight = this.rubric.weighted ? criterion.weight / 100 : 1;
    const floorPoints = this.rubric.weighted ? parseInt(floorPointsParam) * weight : parseInt(floorPointsParam);
``````

Reply:
Tested this locally and it worked fine. Great idea, thanks!